lib/core/src/display/shape/text/text_field.rs

```diff
@@ -1,4 +1,7 @@
 //! A module defining TextField. TextField is a basegl component displaying editable block of text.
+//FIXME:unused_parens is triggered in this file with no indication of where it happens. It's
+// possibly happening inside shared! macro.
+#![allow(unused_parens)]
```

Comment:
This should be fixed in the macro and needs to be investigated where. Consult with me on Discord - I can help with it.